agent/src/beerocks/slave/son_slave_thread.cpp

```diff
@@ -4301,21 +4301,22 @@ bool slave_thread::handle_autoconfiguration_wsc(Socket *sd, ieee1905_1::CmduMess
         if (cmdu_rx.getNextTlvType() != uint8_t(ieee1905_1::eTlvType::TLV_WSC))
             break;
 
-        auto m2 = cmdu_rx.addClass<ieee1905_1::tlvWscM2>();
-        if (!m2) {
+        auto tlvWscM2 = cmdu_rx.addClass<ieee1905_1::tlvWscM2>();
```

Comment:
Just to understand,
In previous prs you inserted that there is no need to call those variables tlvxyz because it is self explanatory.
What changed? 

Also why the refractor not in separate commit?

Reply:
Because I also had a radio identifier TLV, which was called ruid, and now I needed a string representing the ruid, so I changed my mind and decided to use names which makes it clear these are TLVs.

You're right, I should have split to 2 commits. Now it will be an unnecessary overhead and not that critical for understanding the logic in the change so let's make a one time exception.